index.html

```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
 				</ul>
 			</div>
 		</section>
+		<section class="works">
+			<div class="works-container">
+				<div class="card">
+					<div class="image-container background-dark">
+						<img src="/assets/tuhogar.png" alt="Tu Hogar">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							Tu Hogar a un Clic
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Web page design for provide clients with fast info about the properties that are on sale</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://tuhogaraunclic.com" class="see">See project</a>
+					</div>
+				</div>
+				<div class="card">
+					<div class="image-container background-light">
+						<img src="/assets/elrincon.png" alt="El Rincon">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							El Rincon
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Web page design for the menu, orders and financial administration</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://restaurante-web-eta.vercel.app/" class="see">See project</a>
+					</div>
+				</div>
+				<div class="card">
+					<div class="image-container background-dark">
+						<img src="/assets/farmaciajesus.png" alt="Farmacia Jesus">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							Farmacia Jesus
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Drug store landing page to help clients to find the best prices for their medicine</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://farmaciajesus.onrender.com" class="see">See project</a>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about">
+			<div class="title">
+				<p>About Myself</p>
+			</div>
+			<div class="me position">
+					<p>
+							Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+					</p>
```

Comment:
Good job with using p tag here 👍  But let's fix the font color. You can see the color value below 👍 
<img width="716" alt="Screenshot 2023-03-24 at 11 33 33 PM" src="https://user-images.githubusercontent.com/25887253/227633088-c8272d50-6a96-4c54-9260-9676cfc21027.png">